deploy/olm/23.1.0/bundle.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
The script could be moved up a level and could be made version agnostic.

Reply:
Done.